core/src/test/java/org/apache/accumulo/core/util/compaction/CompactionPrioritizerTest.java

```diff
@@ -28,56 +37,133 @@
+  public void testRootTablePriorities() {
+    assertEquals(ROOT_TABLE_USER.getMinimum(), CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), RootTable.ID, CompactionKind.USER, 0, 0, TABLET_FILE_MAX));
+    assertEquals(ROOT_TABLE_USER.getMinimum() + 1000, CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), RootTable.ID, CompactionKind.USER, 1000, 0, TABLET_FILE_MAX));
+    assertEquals(ROOT_TABLE_USER.getMaximum(), CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), RootTable.ID, CompactionKind.USER, 3000, 0, TABLET_FILE_MAX));
+    assertEquals(ROOT_TABLE_SYSTEM.getMinimum(), CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), RootTable.ID, CompactionKind.SYSTEM, 0, 0, TABLET_FILE_MAX));
+    assertEquals(ROOT_TABLE_SYSTEM.getMinimum() + 1000, CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), RootTable.ID, CompactionKind.SYSTEM, 1000, 0, TABLET_FILE_MAX));
+    assertEquals(ROOT_TABLE_SYSTEM.getMaximum(), CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), RootTable.ID, CompactionKind.SYSTEM, 3000, 0, TABLET_FILE_MAX));
+  }
+
+  @Test
+  public void testMetaTablePriorities() {
+    assertEquals(METADATA_TABLE_USER.getMinimum(), CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), MetadataTable.ID, CompactionKind.USER, 0, 0, TABLET_FILE_MAX));
+    assertEquals(METADATA_TABLE_USER.getMinimum() + 1000, CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), MetadataTable.ID, CompactionKind.USER, 1000, 0, TABLET_FILE_MAX));
+    assertEquals(METADATA_TABLE_USER.getMaximum(), CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), MetadataTable.ID, CompactionKind.USER, 3000, 0, TABLET_FILE_MAX));
+    assertEquals(METADATA_TABLE_SYSTEM.getMinimum(), CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), MetadataTable.ID, CompactionKind.SYSTEM, 0, 0, TABLET_FILE_MAX));
+    assertEquals(METADATA_TABLE_SYSTEM.getMinimum() + 1000,
+        CompactionJobPrioritizer.createPriority(Namespace.ACCUMULO.id(), MetadataTable.ID,
+            CompactionKind.SYSTEM, 1000, 0, TABLET_FILE_MAX));
+    assertEquals(METADATA_TABLE_SYSTEM.getMaximum(),
+        CompactionJobPrioritizer.createPriority(Namespace.ACCUMULO.id(), MetadataTable.ID,
+            CompactionKind.SYSTEM, 3000, 0, TABLET_FILE_MAX));
+  }
+
+  @Test
+  public void testSystemNamespacePriorities() {
+    TableId tid = TableId.of("someOtherSystemTable");
+    assertEquals(SYSTEM_NS_USER.getMinimum(), CompactionJobPrioritizer
+        .createPriority(Namespace.ACCUMULO.id(), tid, CompactionKind.USER, 0, 0, TABLET_FILE_MAX));
+    assertEquals(SYSTEM_NS_USER.getMinimum() + 1000, CompactionJobPrioritizer.createPriority(
+        Namespace.ACCUMULO.id(), tid, CompactionKind.USER, 1000, 0, TABLET_FILE_MAX));
```

Comment:
A lot of test use zero for the num compacting files.  May be good to use something greater than zero to ensure its included in computation.